deployments/local/scripts/create_buckets.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
❓ Do we need to create a Windows version of this script?

Reply:
No, this script runs in the docker container.

Reply:
No, this runs in the Docker container.